docs-guides/source/introductory-quickstart.md

```diff
@@ -184,6 +170,7 @@ An example test image (`daisy.jpg`). Right-click and choose **Save Image** to do
 
 ```python Python
 # Use PIL to load and resize the image to expected size
+import PIL
```

Comment:
I don't think this line is necessary. Is the code ever using `PIL` directly (i.e. using something other than `Image`)?

Reply:
Removed line.